controllers/hibp.js

```diff
@@ -37,6 +37,13 @@ async function notify (req, res) {
     }
   }
 
+  if (breachAlert.IsSpamList || breachAlert.Domain === "" || breachAlert.IsFabricated || !breachAlert.IsVerified) {
+    log.info(`${breachAlert.Name} is fabricated, a spam list, or not associated with a website. \n Breach Alert not sent.`);
```

Comment:
"or is not verified" (but this is just a `log.info()` that goes into the ether and not anything user facing...